SLHCUpgradeSimulations/Configuration/python/customise_PFlow.py

```diff
@@ -55,6 +55,10 @@ def customise_shashlikElectronHOverE( process ) :
         process.ecalDrivenElectronSeeds.SeedConfiguration.maxHOverEBarrel = 0.15 
         process.ecalDrivenElectronSeeds.SeedConfiguration.maxHOverEEndcaps = 0.5 
         process.ecalDrivenElectronSeeds.SeedConfiguration.maxHOverEOuterEndcaps = 1.0
+        if hasattr(process,'ecalDrivenGsfElectrons'):
+            process.ecalDrivenGsfElectrons.hOverEMethodEndcap = cms.int32(3)
+            if hasattr(process,'gsfElectrons'):
```

Comment:
These two `if` statements are "accumulative", so only applied if the previous ones also pass.  Whilst this is generally the case (reconstruction step always has all of `ecalDrivenElectronSeeds`, `ecalDrivenGsfElectrons` and `gsfElectrons`), this could change in future and people could rerun only specific parts of the reconstruction, and hence miss some of the customisation.

I'll submit an amendment where the indentation is the same for all `if` statements in a minute.